datafusion/physical-expr/src/expressions/binary.rs

```diff
@@ -418,6 +419,23 @@ macro_rules! binary_primitive_array_op {
     }};
 }
 
+/// Invoke a compute kernel on a pair of arrays
+/// The binary_primitive_array_op macro only evaluates for primitive types
+/// like integers and floats.
+macro_rules! binary_primitive_array_op_dyn {
+    ($LEFT:expr, $RIGHT:expr, $OP:ident) => {{
+        match $LEFT.data_type() {
+            DataType::Decimal128(_,_) => {
+                Ok(paste::expr! {[<$OP _decimal>]}(&$LEFT, &$RIGHT)?)
+                // Ok(Arc::new($OP(&$LEFT, &$RIGHT).map_err(|err| DataFusionError::ArrowError(err))?))
```

Comment:
is this meant to be left commented out?

Reply:
Aha, forgot to clean it up. I will remove it along with test coverage in next commit. Thanks.